package.json

```diff
@@ -19,26 +19,29 @@
     "bitstamp": "^1.0.4",
     "bl": "^1.2.1",
     "btce": "^0.4.2",
-    "ccxt": "^1.10.119",
     "cexio-api-node": "^1.0.4",
     "cliff": "^0.1.10",
+    "cluster": "0.7.7",
     "codemap": "^1.3.1",
     "colors": "^1.1.2",
     "commander": "^2.9.0",
+    "convnetjs": "^0.3.0",
+    "ccxt": "^1.4.77",
     "forex.analytics": "mkmarek/forex.analytics#7bc278987700d4204e959af17de61495941d1a14",
     "gdax": "^0.4.2",
     "gemini-api": "^2.0.4",
     "glob": "^7.1.1",
     "har-validator": "^5.0.3",
     "idgen": "^2.0.2",
     "kraken-api": "^0.1.7",
+    "mathjs": "^3.16.5",
     "micro-request": "^666.0.10",
     "mime": "^1.4.0",
     "minimist": "^1.2.0",
     "moment": "^2.18.1",
     "mongodb": "^2.2.31",
-    "node-prowl": "^0.1.7",
```

Comment:
I have absolutely no clue how this was missing from my zenbot pull because I pulled literally straight from zenbot.git on carlos8f repo. Either way, I re-added it.